### DISCUSSION TITLE
[Angular 大師之路] 認識 ng-container | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#90] [Announcements]
# [Angular 大師之路] 認識 ng-container | 全端開發人員天梯

今天我們來聊聊在 Angular 上一個有趣又方便，也是個人非常喜歡使用的語法 - &lt;ng-container&gt;

https://fullstackladder.dev/blog/2018/10/28/mastering-angular-13-ng-container/

**Comment by suzijie860706** [comment]
非常實用! 感謝教學
